setup.py

```diff
@@ -7,7 +7,7 @@
     requirements = f.read().splitlines()
 
 setup(name='pitt_broker',
-      version='0.0.2',
+      version='0.1.0',
```

Comment:
Is there a way to ensure the same version is used here in `setup.py` as in `__init__.py`?

Reply:
I've changed the setup file to get the version from `__init__.py` using a regular expression.